test/parser/test_base_parser.rb

```diff
@@ -23,5 +23,80 @@ def test_large_xml
         parser.position < xml.bytesize
       end
     end
+
+    def test_parse_exception_for_multiple_root_elements
```

Comment:
Could you move this to https://github.com/ruby/rexml/blob/master/test/parse/test_element.rb ?

Reply:
I see.
OK.